woof-code/0setup

```diff
@@ -68,7 +68,7 @@ if [ "$1" = "" ] ; then
 fi
 #======================================================================
 
-if wget --help | grep -q '\-\-show\-progress' ; then
+if wget --help | grep -q '--show-progress' ; then
```

Comment:
```
~$ wget --help | grep -q '--show-progress'
grep: unrecognized option '--show-progress'
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```

Reply:
You'll need to be very careful.

Reply:
You need `--` to stop option parsing, i.e. `wget --help | grep -q -- '--show-progress'`

Reply:
Oh, so there _is_ a case for escape.

Reply:
Yes, but you don't need that escaping if you make grep understand this is not an option the right way 😄

Reply:
Also leaving off the '--' worsk

`wget --help | grep -q 'show-progress'`